.github/workflows/ci.yml

```diff
@@ -454,3 +454,20 @@ jobs:
         run: ./ci/pytorch_lightning_integration_tests.sh
       - name: Upload test coverage to Codecov
         uses: codecov/codecov-action@v1.0.12
+
+  onnx_integration_tests:
```

Comment:
Can you change the name to `eval_integration_tests`? Github CI requires unique name for each tasks

Reply:
Oops, yes!